vocab/security/vocabulary.yml

```
@@ -272,6 +272,12 @@ property:
     range: xsd:string
     comment: The challenge property is used to associate a challenge with a proof, for use with a `proofPurpose` such as `authentication`. This string value SHOULD be included in a proof if a `domain` is specified.
 
+  - id: previousProof
+    label: Previous proof
+    domain: sec:Proof
+    range: xsd:anyURI
+    comment: The` previousProof` property is used to identify a proof that MUST be verified before the proof that contains this property.
```

Again, this seems backwards or inverted.
```suggestion
    comment: The `previousProof` property is used to identify a proof that MUST be verified before the proof that contains this property.
```